src/apps/kfone-customer-support/.eslintrc.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Is this the correct license header?

Reply:
This is the one we are using in `WSO2` front end repos.